src/App.js

```diff
@@ -4,25 +4,48 @@ import ContactsList from "./components/ContactsList"
 import ContactsAdd from "./components/ContactsAdd"
 import ContactsView from "./components/ContactsView"
 import "./styles/styles.css"
+import ContactsEdit from "./components/ContactsEdit"
 
 export default function App() {
   const [contacts, setContacts] = useState([])
-  
-  //TODO: Load all contacts on useEffect when component first renders
+  const [update, setupdate] = useState(false)
+
+  useEffect(() => {
+    fetch('http://localhost:4000/contacts')
+      .then(res => res.json())
+      .then(contacts => setContacts(contacts))
+  }, [])
+
+  console.log('main render', contacts)
+  console.log('update', update)
+
+  // Re-renders page when contacts edited or deleted
+  useEffect(() => {
+    fetch('http://localhost:4000/contacts')
+      .then(res => res.json())
+      .then(contacts => setContacts(contacts))
+  }, [update])
```

Comment:
good use of adding update as a dependency so re-render happens having met that condition